src/App.tsx

```diff
@@ -1,15 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [searchValue, setSearchValue] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const movieTitle = movie.title.toLowerCase();
+    const movieDesc = movie.description.toLowerCase();
+    const normalizedSearchValue = searchValue.toLowerCase().trim();
```

Comment:
Let's move this `normalizedSearchValue` out of the filter method.😃
We need it to avoid creating a new variable at each iteration.